api-v2/app/com/foreignlanguagereader/api/client/google/GoogleCloudClient.scala

```diff
@@ -54,21 +57,23 @@ class GoogleCloudClient @Inject()(gcloud: GoogleLanguageServiceClientHolder,
       .setEncodingType(EncodingType.UTF16)
       .build
 
-    withBreakerOption(s"Failed to get tokens from google cloud", Future {
+    withBreaker(s"Failed to get tokens from google cloud", Future {
```

Comment:
Codacy found an issue: [Use pattern matching directly in function definitions](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=6397605)